components/suggest/src/db.rs

```diff
@@ -140,17 +140,17 @@ impl<'a> SuggestDao<'a> {
         {
             (self.conn.prepare_cached(
                 &format!(
-                    "SELECT s.id, k.rank, s.title, s.url, s.provider, NULL as confidence, NULL as keyword_suffix
+                    "SELECT s.id, k.rank, s.title, s.url, s.provider, s.score, NULL as confidence, NULL as keyword_suffix
```

Comment:
i tried to add `IFNULL(s.score, 0.2) AS s.score` here and it wasn't happy with me

Reply:
Try `IFNULL(s.score, 0.2) AS score` 😊

`SELECT s.score` is shorthand for `SELECT s.score AS score`—the left-hand side of the `AS` is the qualified column name, and the right-hand side is the alias; a qualified column name can’t appear on the right-hand side.